pom.xml

```diff
@@ -57,7 +57,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
-            <version>1.27.1</version>
+            <version>1.28.0</version>
         </dependency>
```

Comment:
I assume that this also upgrades `org.apache.commons:commons-lang3`. Please add a comment that this fixes 
 CVE-2025-48924 in `org.apache.commons:commons-lang3:jar:3.16.0:compile`